src/js/common/schemaform/FormPage.jsx

```diff
@@ -102,6 +104,15 @@ class FormPage extends React.Component {
     return { pages: expandedPageList, pageIndex };
   }
 
+  autoSave() {
+    const { form } = this.props;
+    const data = form.data;
+    const { formId, version } = form;
+    const returnUrl = this.props.locationPathname;
```

Comment:
This should be `this.props.location.pathname`.

Reply:
Nice catch, changed.